package.json

```diff
@@ -1,23 +1,4 @@
 {
-  "name": "performance",
-  "version": "1.0.0-beta.2",
-  "private": true,
-  "description": "Performance plugin from the WordPress Performance Group, which is a collection of standalone performance modules.",
-  "author": "The WordPress Contributors",
-  "license": "GPL-2.0-or-later",
-  "keywords": [
-    "performance",
-    "wordpress"
-  ],
-  "homepage": "https://wordpress.org/plugins/performance-lab/",
-  "repository": "git+https://github.com/WordPress/performance.git",
```

Comment:
Same here, `npm install` warns about the lack of `repository`. How about we keep it?

Reply:
Yep, also added it back.